### ISSUE TITLE
Does it support other languages than English?

### ISSUE THREAD

**Issue by chillbert** [lamm-mit/PDF2Audio#2]
I couldn't run the demo on huggingface (Error) - before I install this locally, I wanted to know if it supports other lanugages as well other than just english for TTS?

**Comment by lamm-mit** [comment]
Yes. We’ve added multilingual podcast templates: You can now generate content (from any source language) in French, German, Spanish, Portuguese, Hindi, and Chinese, & others. If you want other languages, or finer control about style, you can also first generate a draft in English and then ask the model to translate to any other language or style using the edit function (e.g.: say "Translate to ..." and then hit "Regenerate Audio..." ).
